sdk/communication/Azure.Communication.Sms/src/SmsClient.cs

```diff
@@ -3,12 +3,14 @@
 
 using System;
 using System.Collections.Generic;
-using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
 using Azure.Core;
 using Azure.Core.Pipeline;
 using Azure.Communication.Pipeline;
+using System.Linq;
```

Comment:
nit: let's sort the usings `System.*` first and then the rest alphabetically

Reply:
https://stackoverflow.com/questions/14060379/import-all-subclasses-like-java-but-in-c-sharp
I dont think we can do System.* in C#? I will sort alphabetically though